.github/pull_request_template.md

```diff
@@ -1,5 +1,14 @@
 <!--
 Hi there! Thanks for submitting a pull request to Concourse!
+
+The title of your pull request will be used to generate the release notes.
+Please make if a brief sentence that describes the PR in the imperative form.
```

Comment:
There's a typo here ("make if"), but this rewording might be better than just fixing the typo:

```suggestion
Please provide a brief sentence that describes the PR, using the [imperative mood].
```

and then add this below the paragraph:

```md
[imperative mood]: https://chris.beams.io/posts/git-commit/#imperative
```